tests/cypress/page-object/site.home.page.ts

```diff
@@ -32,9 +38,25 @@ class SiteHomePage extends BasePage {
             .rightclick()
             .should('have.class', 'context-menu-open')
 
-        this.getIframeBody().contains(this.htmlElements.span, this.elements.textEdit).click()
+        this.getIframeBody().find(this.elements.editSite).click()
         return editPage
     }
+
+    publishSite(site = 'Digitall') {
+        this.getIframeBody()
+            .contains('div[role="row"]', site)
+            .trigger('mouseover') // Stabilize portion right before the right-click so it hover over the right element
+            .rightclick()
+            .should('have.class', 'context-menu-open')
+        this.getIframeBody().find(this.elements.publishSiteinAllLang).click()
+        return workflowDashboard
```

Comment:
Is there a check we can do to see that publish is completed? (checking state is OK)

Reply:
added